### PR TITLE
feat: cart

### PR DESCRIPTION
**Pull request by prakashsakari**
Description:

- Cart page for the e-commerce site, with horizontal product card and vertical price card.

### CODE REVIEW

**Review by Creator-singh — APPROVED** [review]
looks good to me

File: cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    border-bottom: 1px solid var(--accent-color);
+    margin: 1rem 0;
+    padding-bottom: 1rem;
+}
+
+.flex-start {
+    align-self: flex-start;
+}
+
+
```
Comment (aniketsingh13) [inline cart/cart.css:72]: remove spaces

**Review by aniketsingh13 — APPROVED** [review]
look good to me

**Review by Mahi24-prog — COMMENTED** [review]
Navbar is not fully responsive
UI is looking nice

**Pull request merged**